app/controllers/events_controller.rb

```diff
@@ -46,6 +46,7 @@ def update
 
   # DELETE /events/1
   def destroy
+    @event = Event.find(params[:id])
```

Comment:
This is not needed as `#set_event` already memoize it before action, please confirm

Reply:
ok

Reply:
change made